.github/workflows/build.yml

```
@@ -51,27 +51,26 @@ jobs:
         with:
           name: simple64-win64-${{ steps.slug.outputs.sha7 }}.zip
           path: simple64-*.zip
-#  build-mac:
-#    runs-on: macos-10.15
-#    steps:
-#      - uses: actions/checkout@v3
-#      - name: Get short SHA
-#        id: slug
-#        run: echo "::set-output name=sha7::$(git rev-parse --short HEAD)"
-#      - name: brew update
-#        run: |
-#          brew update
-#          brew install hidapi sdl2_net sdl2 qt@5
-#      - name: Run build script
-#        run: |
-#          export qt_version=$(ls /usr/local/Cellar/qt@5)
-#          sudo ln -s /usr/local/Cellar/qt@5/$qt_version/plugins /usr/local/plugins
-#          sudo ln -s /usr/local/Cellar/qt@5/$qt_version/mkspecs /usr/local/mkspecs
-#          sudo ln -s /usr/local/Cellar/qt@5/$qt_version/bin/qmake /usr/local/bin/qmake
-#          export PATH=$PATH:/usr/local/Cellar/qt@5/$qt_version/bin
-#          sudo sh ./build.sh
-#      - name: Upload dmg
-#        uses: actions/upload-artifact@v3
-#        with:
-#          name: simple64-osx-${{ steps.slug.outputs.sha7 }}.dmg
-#          path: simple64/simple64-*.dmg
+  build-mac:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get short SHA
+        id: slug
+        run: echo "::set-output name=sha7::$(git rev-parse --short HEAD)"
+      - name: brew update
+        run: |
+          brew update
+          brew install hidapi sdl2_net sdl2 qt@6
+      - name: Run build script
+        run: |
+          export qt_version=$(ls /usr/local/Cellar/qt)
+          sudo ln -s /usr/local/Cellar/qt/$qt_version/plugins /usr/local/plugins
+          sudo ln -s /usr/local/Cellar/qt/$qt_version/mkspecs /usr/local/mkspecs
+          export PATH=$PATH:/usr/local/Cellar/qt/$qt_version/bin
```

```suggestion
          export qt_prefix="$(brew --prefix qt)"
          sudo ln -s "$qt_prefix/plugins" /usr/local/plugins
          sudo ln -s "$qt_prefix/mkspecs" /usr/local/mkspecs
          export PATH="$PATH:$qt_prefix/bin"
```
`Cellar` paths are an implementation detail. You should not rely on them.

---

There hasn't been a MacOS build for a long time so that will be old code anyway.

---

I can see that, but perhaps it would be good for this PR to stop carrying forward mistakes in old code, no?

---

These mistakes are likely also present in the Mupen64Plus code from which this is based off.
So they've been there probably forever XD.

Sadly the build failed so some more investigation will be needed (I don't own a Mac though).

---

The build is failing because the Qt provided by brew on Mac doesn't support Vulkan. I opened a PR for this (https://github.com/Homebrew/homebrew-core/pull/111474) but I'm not sure if it will be merged

---

I had a quick look at that, I hop eit will be merged but I felt just from reading the comments that they didn't seem very friendly - might be just me though.

---

I got that same vibe:) but I'm sure I've come across that way plenty of times, hopefully something will get merged to provide Vulkan support, even if it's not that.